README.md

```diff
@@ -2,17 +2,22 @@
 
 The new home of the betanextbest.wellcomecollection.org website.
 
-# client
-
-    cd client
-    npm install
-    npm run compile OR npm run compile:watch (for developing)
-       
-# server
-    
-    cd server
-    npm install
-    npm run watch
-    
+## client
+
+    - `cd client`
```

Comment:
I am talking about the `-` not the `.
Also in Markdown, and indented block like that is a codeblock.

To be more explicit you could have:
<pre>
```bash
# bashcode
```
</pre>